### PR TITLE
Update README file for formatting fixes

### PR DESCRIPTION
**Pull request by trbehera**
Fix some of the formatting issues in the documents.

Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

File: docs/linux.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ To run the device against the SCT for the DI protocol, do the following:
   $ ./build/linux-client
   ```
 
-## 7. Compiling and runing of unit tests for SDO
+## 7. Compiling and Runing of Unit Tests for SDO
```
Comment (Darshini-Parikh93) [inline docs/linux.md:75]: Spell Check: Runing -> Running

File: docs/tpm.md [hunk 2]
```diff
@@ -39,11 +39,11 @@ $ ./install_tpm_libs.sh -i
...
 $ ./install_tpm_libs.sh -u
 ```
-> **Note:** Installation of these components may require elevated permissions. Please use 'sudo' to execute the script.
+> ***Note:*** Installation of these components may require elevated permissions. Please use 'sudo' to execute the script.
 
 ### 2.1 Building and Installing Libraries for Trusted Platform Module (TPM)
 
-Following steps should be performed if SDO TPM [TPM-Library-Installation-Script](../utils/install_tpm_libs.sh) script is not used to setup SDO TPM libraries. Install only tpm2-tss library to enable TPM enabled SDO Client-SDK code compilation. To enable compilation and execution of TPM enabled SDO Client-SDK code install all three libraries namely: tpm2-tss, tpm2-abrmd, tpm2-tools and tpm2-tss-engine
+Following steps should be performed if SDO TPM [TPM-Library-Installation-Script](../utils/install_tpm_libs.sh) script is not used to setup SDO TPM libraries. Install only tpm2-tss library to enable TPM enabled SDO Client-SDK code compilation. To enable compilation and execution of TPM enabled SDO Client-SDK code install all three libraries namely: tpm2-tss, tpm2-abrmd, tpm2-tools, and tpm2-tss-engine
```
Comment (prateekcr) [inline docs/tpm.md:46]: Comma after 'TPM enabled SDO Client-SDK code' and full-stop after 'tpm2-tss-engine' in the last statement?

File: docs/tpm.md [hunk 3]
```diff
@@ -273,7 +273,7 @@ Find a persistent storage index that is unused in the TPM and note it down. It u
 # 5. Issue the command 
 
   $ make 
-  (You may need to run �sudo apt install make gcc� before running this command successfully).
+  (You may need to run sudo apt install make gcc before running this command successfully).
```
Comment (Darshini-Parikh93) [inline docs/tpm.md:276]: Rendering issue of some special character before and after `sudo apt install make gcc` ??

File: docs/linux.md (hunk 1 above)
Comment (trbehera) [reply docs/linux.md:74]: Done.

File: docs/tpm.md (hunk 2 above)
Comment (trbehera) [reply docs/tpm.md]: Done.

File: docs/tpm.md (hunk 3 above)
Comment (trbehera) [reply docs/tpm.md]: Done

**Review by Darshini-Parikh93 — APPROVED** [review]

File: docs/mbedos.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
-- SanDisk* 8 or 16GB microSD memory card.
+- Serial Peripheral Interface (SPI)-based microSD* card breakout board.
+- SanDisk* 8 or 16GB microSD memory card.
 - Ethernet cable.
 - Jumper wires.
 - MICROUSB cable.
 
-## 2. Software requirements:
+## 2. Software Requirements:
 - Linux* OS (Ubuntu* OS version 18.04 LTS) as host machine.
```
Comment (DukeDavis12) [inline docs/mbedos.md:12]: Is this one missed out ?

**Comment by trbehera** [comment]
Updated further via PR #63

**Pull request closed without merging**